docs/introduction/en/basics.md

```diff
@@ -0,0 +1,109 @@
+execution of transaction.
+* `gas`: A scalar value indicating the total amount of gas paid for the cost of the execution of transaction. This is paid up-front,
+before any actual computation is done, and may not be increased or refunded later. It is
+the transaction sender’s responsibility to avoid any extravagance caused by an unnecessarily high gasLimit.
+* `value`: A scalar value equal to the amount of Drip that the transactions sender wants to transfer to the recipient, i.e. the
+account specified in transaction or the newly created contract. 
+* `storageLimit`: A scalar value indicating the maximum increment of storage used in the execution of transaction, measured in
+bytes.
+* `epochHeight`: A scalar value specifying the range of epochs where transaction can be executed. Such that transaction can only be executed between the epochs of [Te − 100000, Te + 100000].
+* `chainId`: A binary chain id indicating where transaction is intended to be executed. 
+* `data`: A byte array specifying the input data of the message call to an existing contract. 
+* `v`, `r`, `s`: Corresponding fields of the recoverable ECDSA signature of transaction.
+
+
+### Transaction's stage
+An transaction is composed by external account, and sent to a node by calling its `cfx_sendRawTransaction` RPC method. After sending, the transaction will go through several different states:
+
+1. `Accepted by the RPC`: After the provider received the cfx_sendRawTransaction RPC call, it would attempt to do basic verification and `insert it into the transaction pool`. If there is an obvious error of the transaction, e.g., RLP decoding error, signature verification error, it will be rejected immediately. Otherwise, it will be inserted into the transaction pool and wait to be mined, and the RPC will return a transaction hash of the transaction.
+2. `Wait to be mined`: A Conflux miner will choose valid transactions from its pool, whose nonce matches the `expected one` and the balance is enough to pay the `value + gas * gasPrice + storage_limit * (10^18/1024)` that will be mined in the new block. If one transaction does not meet these two requirements, then it will get stucked in the transaction tool.
+3. `Mined`: After mining, a tx will not be executed immediately, Conflux has a defer execution mechanism, all tx will be deferred 5 epoch to execute.
+4. `Executed (success or failed)`: Not all executed transactions have succeeded, some have failed. You can check this out by the transaction's `status` or transaction receipt's `outcomeStatus`. `0` means success, `1` means failure.
+5. `Confirmed`: If a transaction is executed successfully, it's status may be reverted, possibly due to the pivot chain switching. Only after it is confirmed, it's status is finalized. By checking one transaction's block epoch number with the latest confirmed epoch number, you can determine whether a transaction is confirmed.
```

Comment:
If a transaction has been executed successfully, it may still be reverted, possibly due to a pivot chain reorganization. A transaction's status is only finalized once it has been confirmed. You can determine whether a transaction is confirmed by comparing its epoch number to the latest confirmed epoch number.